src/lib/select/select.spec.ts

```diff
@@ -206,6 +207,114 @@ describe('MdSelect', () => {
 
   });
 
+  describe('forms integration', () => {
+    let fixture: ComponentFixture<BasicSelect>;
+    let trigger: HTMLElement;
+
+    beforeEach(() => {
+      fixture = TestBed.createComponent(BasicSelect);
+      fixture.detectChanges();
+
+      trigger = fixture.debugElement.query(By.css('.md-select-trigger')).nativeElement;
+    });
+
+    it('should set the view value from the form', () => {
+      let value = fixture.debugElement.query(By.css('.md-select-value'));
+      expect(value).toBeNull('Expected trigger to start with empty value.');
+
+      fixture.componentInstance.control.setValue('pizza-1');
+      fixture.detectChanges();
+
+      value = fixture.debugElement.query(By.css('.md-select-value'));
+      expect(value.nativeElement.textContent).toContain('Pizza',
```

Comment:
Prefer breaking at the higher syntactic level, e.g.:
```ts
expect(value.nativeElement.textContent)
    .toContain('Pizza', `Expected trigger to be populated by the control's new value.`);
```
(general rule-of-thumb that makes code more readable in all languages)